### ISSUE TITLE
Nodemon + Node20 => `TypeError [ERR_INVALID_ARG_TYPE]: The "code" argument must be of type number. Received type string ('SIGUSR2')`

### ISSUE THREAD

**Issue by cyrilchapon** [prisma/prisma#19151]
### Bug description

After upgrading Node and my deps; I started getting 

> TypeError [ERR_INVALID_ARG_TYPE]: The "code" argument must be of type number. Received type string ('SIGUSR2')

when nodemon is restarting my process.

Here is the full stacktrace with `DEBUG="*"` :

```
rs
  nodemon bus emit: restart +10s
  nodemon:run sending kill signal to 1261 +10s
  nodemon:run checking ps tree for pids of 1259 +2ms
  nodemon:run bus.emit(exit) via SIGUSR2 +3ms
  nodemon bus emit: exit +48ms
[nodemon] starting `dotenv -- node --loader @bleed-believer/path-alias src/serve.ts`
  prisma:client:libraryEngine:exitHooks  exit event received: SIGUSR2 +0ms
  prisma:client:libraryEngine:exitHooks  exit event received: exit +1ms
  nodemon:run spawn sh -c dotenv -- node --loader @bleed-believer/path-alias src/serve.ts +4ms
  nodemon bus new listener: exit (1) +4ms
  nodemon:run start watch on: [ '*.*', re: /.*\..*/ ] +0ms
  nodemon:watch early exit on watch, still watching (1) +10s
node:internal/errors:490
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "code" argument must be of type number. Received type string ('SIGUSR2')
    at process.set [as exitCode] (node:internal/bootstrap/node:124:9)
    at process.exit (node:internal/process/per_thread:188:24)
    at process.<anonymous> (/Users/cyrilchapon/Code/wib/mtn/mtn-api/node_modules/@prisma/client/runtime/library.js:99:2170)
    at Object.onceWrapper (node:events:626:26)
    at process.emit (node:events:511:28) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

### How to reproduce

- Run a script importing prisma from nodemon
- Type `rs` in terminal

### Expected behavior

I'd expect it not to change behavior from when using Node 18

### Prisma information

Irrelevant

### Environment & setup

- OS: macOS
- Database: PostgreSQL
- Node.js version: v20.1.0

Additional info :
- "nodemon": "^2.0.21"
- "ts-node": "^10.9.1"

### Prisma Version

```
prisma                  : 4.13.0
@prisma/client          : 4.13.0
Current platform        : darwin
Query Engine (Node-API) : libquery-engine 1e7af066ee9cb95cf3a403c78d9aab3e6b04f37a (at ../../node_modules/@prisma/engines/libquery_engine-darwin.dylib.node)
Migration Engine        : migration-engine-cli 1e7af066ee9cb95cf3a403c78d9aab3e6b04f37a (at ../../node_modules/@prisma/engines/migration-engine-darwin)
Format Wasm             : @prisma/prisma-fmt-wasm 4.13.0-50.1e7af066ee9cb95cf3a403c78d9aab3e6b04f37a
Default Engines Hash    : 1e7af066ee9cb95cf3a403c78d9aab3e6b04f37a
Studio                  : 0.484.0
Preview Features        : postgresqlExtensions, clientExtensions
```

**Comment by cyrilchapon** [comment]
Oh my bad. I thought v20 was actually current; misread.

It was working perfectly under Node 18

**Comment by Jolg42** [comment]
Nice! I'll keep this open, we will check this later with v20 once we start fully testing against it.

**Comment by Jolg42** [comment]
@SevInf told me this should already be fixed with Prisma `4.14.0`

**Comment by cyrilchapon** [comment]
> @SevInf told me this should already be fixed with Prisma 4.14.0

That's indeed the case 🤩 Confirmed by upgrading.
Closing !
